> The feature test in the new version failed. log:
> 
> ```
> E:\feature-test\vcpkg\buildtrees\google-cloud-cpp\x86-windows-dbg\google\cloud\channel\google/cloud/channel/v1/common.pb.h(544): error C2059: syntax error: 'constant'
> E:\feature-test\vcpkg\buildtrees\google-cloud-cpp\x86-windows-dbg\google\cloud\channel\google/cloud/channel/v1/common.pb.h(545): error C2238: unexpected token(s) preceding ';'
> ```

Ugh, that code looks like so:

```
  static constexpr CustomerType DOMAIN =
    CloudIdentityInfo_CustomerType_DOMAIN;
```

Looks like `DOMAIN` is a macro on Windows, and we are going to need to fix  Protobuf to do something like this:

https://github.com/protocolbuffers/protobuf/blob/aa22c5e0de189b74271335eea179e5321d88768e/src/google/protobuf/port_def.inc#L885-L888

I can submit such a fix to Protobuf, but it will be weeks before a release contains the fixes, and then more time to pick it up here.  I think I need to sprinkle a few `"support": "!windows"` in these features.  I should be able to get that done today.

--- a/ports/google-cloud-cpp/vcpkg.json
+++ b/ports/google-cloud-cpp/vcpkg.json
@@ -107,7 +107,8 @@
             "grpc-common"
           ]
         }
-      ]
+      ],
+      "supports": "!windows"
     },
     "assuredworkloads": {
       "description": "Assured Workloads API C++ Client Library",
@@ -191,7 +192,8 @@
             "grpc-common"
           ]
         }
-      ]
+      ],
+      "supports": "!windows"
     },
     "cloudbuild": {
       "description": "Cloud Build API C++ Client Library",

--- a/ports/google-cloud-cpp/vcpkg.json
+++ b/ports/google-cloud-cpp/vcpkg.json
@@ -99,6 +99,7 @@
     },
     "asset": {
       "description": "Cloud Asset API C++ Client Library",
+      "supports": "!windows",
       "dependencies": [
         {
           "name": "google-cloud-cpp",
@@ -107,8 +108,7 @@
             "grpc-common"
           ]
         }
-      ],
-      "supports": "!windows"
+      ]
     },
     "assuredworkloads": {
       "description": "Assured Workloads API C++ Client Library",
@@ -184,6 +184,7 @@
     },
     "channel": {
       "description": "Cloud Channel API C++ Client Library",
+      "supports": "!windows",
       "dependencies": [
         {
           "name": "google-cloud-cpp",
@@ -192,8 +193,7 @@
             "grpc-common"
           ]
         }
-      ],
-      "supports": "!windows"
+      ]
     },
     "cloudbuild": {
       "description": "Cloud Build API C++ Client Library",

--- a/versions/g-/google-cloud-cpp.json
+++ b/versions/g-/google-cloud-cpp.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "a3ba89c66f66ee89018f5a7890ee34404c49489c",
+      "git-tree": "71a72e78da707ee567b96bc835cf25cd8e8eb975",
       "version": "2.0.0",
       "port-version": 0
     },

